tracer/src/Datadog.Trace/Logging/LogEnricher.cs

```diff
@@ -25,21 +25,21 @@ public LogEnricher(ILogProvider logProvider)
             _logProvider = logProvider;
         }
 
-        public void Initialize(Tracer tracer)
+        public void Initialize(IScopeManager scopeManager, string defaultServiceName, string version, string env)
```

Comment:
I like the symmetry between this Initialize and the `LibLogScopeEventSubscriber.ctor()`! 😍